src/cosmosdb-preview/linter_exclusions.yml

```diff
@@ -24,6 +24,9 @@ cosmosdb create:
     gremlin_databases_to_restore:
       rule_exclusions:
       - option_length_too_long
+    enable_materialized_views:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Instead of adding exclusions, can you add a short option for this parameter? For example `--enable-mv`

Reply:
We want to keep the parameter name same across all the az interfaces (rest, cli and power shell) so that it does not confuse the customers.

Reply:
I mean adding `options_list=['--enable-materialized-views', '--enable-mv']`. It won't break consistency, just add one more option